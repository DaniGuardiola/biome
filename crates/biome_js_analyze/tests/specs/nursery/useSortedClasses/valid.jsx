<div class="p-4 px-4" />;
