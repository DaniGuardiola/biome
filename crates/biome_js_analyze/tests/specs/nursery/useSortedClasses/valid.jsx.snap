---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: valid.jsx
---
# Input
```jsx
<div class="p-4 px-4" />;

```


